### ISSUE TITLE
missing AutonomousDatabase class in oci/database.py

### ISSUE THREAD

**Issue by clayms** [mingrammer/diagrams#467]
https://github.com/mingrammer/diagrams/blob/bc14aec21cbe025c9e5b87b3da36d9ed7d12b426/diagrams/oci/database.py#L85

should be `Autonomous`

https://github.com/mingrammer/diagrams/blob/bc14aec21cbe025c9e5b87b3da36d9ed7d12b426/diagrams/oci/database.py#L15

`AutonomousWhite` will also have to be fixed.

https://github.com/mingrammer/diagrams/blob/bc14aec21cbe025c9e5b87b3da36d9ed7d12b426/diagrams/oci/database.py#L86

https://github.com/mingrammer/diagrams/blob/bc14aec21cbe025c9e5b87b3da36d9ed7d12b426/diagrams/oci/database.py#L11

```
NameError                                 Traceback (most recent call last)
~/test.py in ()
      581             if "__" not in str(x)])
      582     for grp in diag_res_grps:
----> 583         exec(f"import diagrams.{res}.{grp}")
      584         grp_nodes = [
      585             x[0] for x in eval(f"inspect.getmembers(diagrams.{res}.{grp}, inspect.isclass)")

~/test.py in ()

/home/user/miniconda/envs/test/lib/python3.6/site-packages/diagrams/oci/database.py in <module>()
     83 # Aliases
     84 
---> 85 ADB = AutonomousDatabase
     86 ADBWhite = AutonomousDatabaseWhite
     87 DBService = Databaseservice

NameError: name 'AutonomousDatabase' is not defined
```

**Comment by clayms** [comment]
also these:

https://github.com/mingrammer/diagrams/blob/bc14aec21cbe025c9e5b87b3da36d9ed7d12b426/diagrams/oci/database.py#L87

https://github.com/mingrammer/diagrams/blob/bc14aec21cbe025c9e5b87b3da36d9ed7d12b426/diagrams/oci/database.py#L88

**Comment by mingrammer** [comment]
It was fixed in v0.19.1. Thanks for reporting it :)
